pages/api/register.js

```diff
@@ -6,15 +6,15 @@ const prisma = new PrismaClient()
 
 export default async function handler (request, response) {
   if (request.method === 'POST') {
-  // Process a POST request
+    // Process a POST request
     const { fullName, email, password } = request.body
     const user = await prisma.user.findFirst({
       where: {
         email: email
       }
     })
     if (user) {
-      response.status(302).json({ message: 'User Already exists' })
+      response.send(406).json({ message: 'User Already exists' })
     } else {
```

Comment:
I think you can remove this since the prisma client will just throw an error if the user already exists. not required to do a check here